README.md

```diff
@@ -624,6 +622,19 @@ WriteRequest{
 
 ### Privacy Considerations
 
+The `did:earth` method is a fully compliant IID and DID. As such the privacy considerations taken into account are applicable to this method. The reader is directed towards the privacy considerations discussed in [section 10 of the W3C DID Core specification](https://w3c.github.io/did-core/#privacy-considerations).
+
+#### Linked Resources
+
+Linked resources within IIDs support privacy in three key three key ways:
+
+1.  Avoids posting potentially sensitive information on-chain in an unavoidably public and irrevocable manner.
+2.  Provides a service endpoint that can apply appropriate privacy and security checks before revealing information.
+3.  The hashgraph resource descriptor type obscures not only the content of the linked resource, but also the quantity of resource objects.
+
+
+> Note: Not sure where this JSON goes. Feels hanging?
```

Comment:
That should be in the linked resources definition section, rather than the privacy considerations section.

Reply:
Actually, I think we can just remove this. The current linked resources section contains this proof object within the larger linkedResource json:

```javascript
{
	"linkedResource": [{

		"path"(optional): // IID Resource path for this resource in the asset namespace, e.g., /myResource.png

			"id"(optional): // IID Reference ID for this resource, e.g., #myResource.png

			"rel"(optional): // the relationship of this resource to the IID Asset

			"type": "nft:ResourceDescriptor", // The JSON-LD type of this resource

		"proof": [ // an array of proofs, any of which may be used
			{
				"type"(optional): "hash" | "hashgraph" | "hashset" // the form of proof used to verify the resource as authentic
				"stage"(optional): "raw" | "compressed" | "encrypted" | "encoded" // the 
				"value"(optional): hash | hashgraph | hashset, // the actual proof for this particular resource
			}
		],
		"resourceFormat": media type, // the IANA media-type of the linked resource

		"compression"(optional): "gzip" | "none", // the compression (performed on the raw resource) of an inline resource

		"encryption": [open ended
			for arbitrary extensibility
		], // the encryption technique applied after compression and before encoding

		"encoding"(optional): "native" | "multibase" | "string", // the encoding (after compression and encryption) of an inline resource; "native" means the inline resource is native JSON-LD with neither compression nor encryption

		"endpoint"(optional): url, // a URL at which this resource can be retrieved before decrypting and decrompressing

		"resource"(optional): "..."
		a representation of the resource
		for inline
		distribution,
		encoded as per "encoding"
    }]
}
```

Anyreason to include the proof json on its own aswell?